app/file.go

```diff
@@ -728,8 +729,11 @@ func (t *uploadFileTask) postprocessImage() {
 		var err error
 		decoded, typ, err = image.Decode(t.newReader())
 		if err != nil {
-			mlog.Error(fmt.Sprintf("Unable to decode image err=%v", err))
-			return
+			decoded, err = tiff.Decode(t.Input)
```

Comment:
Did you verify that `image.Decode` fails to recognize a TIFF stream? 3/5 Seems expensive to attempt to TIFF-decode on every other error if it can be avoided. Otherwise +1.

Reply:
Hi @levb - `image.Decode` does recognize a TIFF stream. I found the preview didn't show when I removed TIFF from constants in utils/constants in the webapp. I can close this PR and open the PR on the webapp and add the TIFF constant to redux.

Reply:
Right, so my recommendation is for you to restore the original lines #730-#732, since the code there should not be TIF-specific. `image.Decode` does not know anything about the filename/extension nor the mime type, it detects the image type by reading and analyzing the headers.

I also think that both `.TIF` amd `.TIFF` should be added in model.go, as you first had them.